### ISSUE TITLE
[a11y] Cannot run report/label from a record set

### ISSUE THREAD

**Issue by grantfitzsimmons** [specify/specify7#1366]
**Describe the bug**
Reports and labels cannot be run from record sets.

**To Reproduce**
1. Go to Reports
2. Select a report/label
3. See error

**Expected behavior**
![image](https://user-images.githubusercontent.com/37256050/165996461-f46cf489-7dea-42fe-bbaa-3d7ef8107d9c.png)

**Screenshots**
![image](https://user-images.githubusercontent.com/37256050/165996417-2cc2a929-dea6-4556-a685-1902b9577f58.png)

**Desktop (please complete the following information):**
 - OS: macOS, Edge

**Comment by maxxxxxdlp** [comment]
Duplicate of #1276
